src/test/scala/wolfendale/nunjucks/ParserSpec.scala

```diff
@@ -90,4 +90,11 @@ class ParserSpec extends FreeSpec with MustMatchers with OptionValues {
 
     println(s"!$result!")
   }
+
+  "asdf" in {
+
+    val tester = new ExpressionTester()
+
+    println(tester.ast("[[1, 2], [1, 2]]"))
```

Comment:
Do any of the tests in this file want to actually have assertions?

Reply:
It was mostly just used as a playground, it doesn't contain any useful tests. Should probably be deleted. I'll remove it and another one in a separate commit.